docs/User-Research/Winter 2021, Round 4 Synthesis.md

```diff
@@ -0,0 +1,166 @@
+# Winter 2021, Round 4 Synthesis 
+
+[Issue #669](https://github.com/raft-tech/TANF-app/issues/669)
+
+---
+
+**Table of Contents:**
+
+- [What We Wanted to Learn](#what-we-wanted-to-learn)
+- [Whom We Talked To](#whom-we-talked-to)
+- [What We Learned (Top Insights)](#what-we-learned-top-insights)
+- [What We Learned (Additonal insights, hypotheses, and validations)](#what-we-learned-additional-hypotheses-and-validations)
+- [What's Next](#whats-next)
+- [Footnotes](#footnotes)
+
+---
+
+## What We Wanted to Learn 
+
+**Our goals for this research were to:**
+- Prioritize the pain points we identified from previous rounds for urgency and immediacy 
+- Cross-reference interlocking pain points between OFA and STTs
+- Narrow our current set of personas to the most comprehensive primary personas so that the journey map and, consequently, our STT MVP is scoped to include the most pressing organizational needs at this time 
+
+**Our research questions were:**
+1. Which gaps in knowledge in our current state journey maps are high priority?
+2. Which STT pain points take priority for the upcoming STT MVP? What do we want to test?
+3. Are there OFA pain points and STT pain points overlapping and therefore more efficient to solve together?
+4. Which personas have the most different experiences? How can we be sure we’ve covered the most diversity?
+5. Are there behavioral changes implicated in the data process map that we can address in the target state or aspirational state maps?
+
+**We also added two sprint responsive research questions:**
+
+6. What, if any, loose ends exist for OFA MVP which might be revealed by usability testing and design auditing?
+7. What insight might regional managers and OFA staff provide to engaging STT Users in on-going user research? 
+
+---
+
+## Whom We Talked To
+**This round, we talked to the OFA DIGIT team and Regional Offices, and five participants from States that:**
+
+- Had not participated in past research
+- Submit Universe Data
+- Submit Sample Data
+- Have caseloads either less than 3,500 per month or greater than 9,500
+
+---
+
+## What We Learned (Top Insights)
+
+Included in the learnings of this round is that our participants prefered to be called "Data Analysts" over "Data Preppers", and we will use that convention from here on in.
+
+
+### Analysts value data cleansing/validation that allows them to predict whether their submission will be accepted
+>Related to research questions 3 and 5
+
+One ST data analyst discussed that he has a program from a third party that cleanses and validates the data for him before he submits it, and it “predicts whether or not a submission would be accepted.” Other analysts indicated an interest in having this ability. 
+
+**Project Imapacts**
+- We will plan an interview with the vendor of the program and a look at the software application to leverage a competitive analysis with HMDA's data cleansing process in designing our solution.`  
```

Comment:
👍🏾 👍🏾 interested to know if there's any possibility of moving up the check prior to clicking submit.